### ISSUE TITLE
[BUG] Fix flaky test org.opensearch.clustermanager.ClusterManagerTaskThrottlingIT.testTimeoutWhileThrottling

### ISSUE THREAD

**Issue by Rishikesh1159** [opensearch-project/OpenSearch#5452]
**Describe the bug**
New flaky test `org.opensearch.clustermanager.ClusterManagerTaskThrottlingIT.testTimeoutWhileThrottling`

```
org.opensearch.clustermanager.ClusterManagerTaskThrottlingIT > testTimeoutWhileThrottling FAILED
    java.lang.AssertionError: expected:<1> but was:<0>
        at __randomizedtesting.SeedInfo.seed([2062CE3E58A80654:BE99C32BBA65E511]:0)
        at org.junit.Assert.fail(Assert.java:89)
        at org.junit.Assert.failNotEquals(Assert.java:835)
        at org.junit.Assert.assertEquals(Assert.java:647)
        at org.junit.Assert.assertEquals(Assert.java:633)
        at org.opensearch.clustermanager.ClusterManagerTaskThrottlingIT.testTimeoutWhileThrottling(ClusterManagerTaskThrottlingIT.java:175)
```

From [comment](https://github.com/opensearch-project/OpenSearch/issues/1715#issuecomment-1335981772)

Found: 
-> https://build.ci.opensearch.org/job/gradle-check/6742/
-> https://build.ci.opensearch.org/job/gradle-check/6556/

**To Reproduce**

**Expected behavior**

**Plugins**
Please list all plugins currently enabled.

**Screenshots**

**Host/Environment (please complete the following information):**

**Additional context**

**Comment by andrross** [comment]
@Rishikesh1159 I believe this was fixed in #5152

Do you have an example of a more recent failure?

**Comment by Rishikesh1159** [comment]
Nice, I didn't know we already fixed it. Last occurrence I see this is: https://build.ci.opensearch.org/job/gradle-check/6742. Thanks @andrross for pointing out. I think this is duplicate of already closed issue. I can close this one then
